contributors/design-proposals/api-proposal-design-template.md

```diff
@@ -0,0 +1,84 @@
+# Description of your Api
+
+## Owners
+
+Sig owner: list of sigs that will own the design, implementation, and support
+Individual owners: list of individual GitHub users that will own the design, implementation, and support
```

Comment:
SGTM.  I will do that as a follow up.  The feature template is in bad need of an overhaul anyway